search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -160,20 +163,16 @@ internal fun buildTemporalQuery(params: MultiValueMap<String, String>): Temporal
     if (timerelParam == "between" && endTimeParam == null)
         throw BadRequestDataException("'endTime' request parameter is mandatory if 'timerel' is 'between'")
 
-    val (timerel, time) = if (timerelParam == null && timeParam == null) {
```

Comment:
Not sure the code is easier to read and maintain in this new version...

Reply:
it should be better now